### DISCUSSION TITLE
Applying Different Category Tags to subsequent scans

### DISCUSSION THREAD

**Discussion by mantacid** [cea-sec/ivre#1599] [Q&A]
When I did my first scan, everything went normally. When I did my second scan, however, The category and source tags I gave to the results of that scan applied themselves to all the results of the previous scan. How do I stop this from happening?

**Comment by mantacid** [comment]
I just realized that the second time I ran the scan I pulled from scans/ROUTABLE/up. Issue solved, I'm just new to this.

**Reply by p-l-** [reply]
OK great, happy to read you've fixed your issue!
